src/main/java/chessmaster/game/ChessBoard.java

```diff
@@ -372,6 +375,44 @@ public ChessBoard toBoard(String board) {
         return new ChessBoard(this.playerColor, boardTiles);
     }
 
+    //@@author ken-ruster
+    public void executeMoveArray(ArrayList<String> moves, Human human, CPU cpu) throws ChessMasterException {
+        boolean isPlayersTurn = playerColor.isWhite();
+
+        for (String move : moves) {
+            String[] moveCommandArray = move.split(" ");
```

Comment:
Change this delimiter to `\\s+` just in case the user changes the storage file to have multiple spaces